app/models/work_group.rb

```diff
@@ -71,6 +71,14 @@ def minified_url(user)
     end
   end
 
+  def assign_colors()
+    # orange, yellow, green, blue, pink, purple
+    colors = ['#FF5733', '#FFE632', '#11841F', '#2B86E7', '#E72BD8', '#B554FF']
```

Comment:
I hard-coded colors for now...not sure how many colors we can have on the whiteboard? I could also create a function where it creates a dark color by randomly selecting hex values.

Reply:
hardcoding should be fine for now